physionet-django/console/templates/console/training_type/index.html

```diff
@@ -0,0 +1,74 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}Training Types{% endblock %}
+
+{% block content %}
+ <div class="card mb-3">
+  <div class="card-header">
+    Training Types <span class="badge badge-pill badge-info">{{ training_types|length }}</span>
+  </div>
+  <div class="card-body">
+    <div><button type="button" class="btn btn-sm btn-danger" data-toggle="modal" data-target="#op-training">Create/Update On Platform Training</button></div>
+    <div class="table-responsive">
+      <table class="table table-bordered">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Valid Duration</th>
+            <th>On Platform Training</th>
```

Comment:
I agree, we should be removing this. Because the tab on console says "OP Training", i cant think of a case where, we would have a non OP training here